cookbook/tools/scrapegraph_tools.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+from phi.agent import Agent
+from phi.tools.scrapegraph_tools import ScrapeGraphTools
+
+api_key = os.getenv("SGAI_API_KEY")
```

Comment:
you can remove this after adding it to the class

Reply:
Done